Branch-SDK/Branch-SDK/BNCServerInterface.m

```diff
@@ -121,12 +121,10 @@ - (NSURLRequest *)prepareGetRequest:(NSDictionary *)params url:(NSString *)url r
     fullParamDict[@"retryNumber"] = @(retryNumber);
     
     NSString *appId = [BNCPreferenceHelper getAppKey];
-    // TODO re-add this
-    //    NSString *branchKey = [BNCPreferenceHelper getBranchKey];
-    //    if (![branchKey isEqualToString:NO_STRING_VALUE]) {
-    //        fullParamDict[KEY_BRANCH_KEY] = branchKey;
-    //    }
-    //    else if (![appId isEqualToString:NO_STRING_VALUE]) {
```

Comment:
Note that this was an `else if` here -- Dmitri had requested that we only send one value or the other.